CHANGES.md

```diff
@@ -1,3 +1,12 @@
+# 0.20.1
```

Comment:
I think this should be release 0.21 as we break backwards compatibility. That's still not proper semantic versioning but at least it's clearer that we break things than a .1 release, which is really reserved for bugfixes only.